Docs upgrade (#3370)
modrinth/code: apps/labrinth/README.md

![labrinth banner](https://user-images.githubusercontent.com/12068027/100479891-d6bab300-30ac-11eb-8336-b4cad376a03d.png)

## Modrinth API (Labrinth)

> Read more about the API and how to use it on the [Labrinth API documentation page](https://docs.modrinth.com/api/).

Reviewer: ```suggestion
Read more about the API and how to use it in the [API spec](https://docs.modrinth.com/api/).
```
